lib/common/Loading.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class Loading extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        margin: EdgeInsets.only(top: 400),
```

Comment:
Why are giving it margin from top? It is not a good practice to give such huge margin